Dev 877 update readme md to match new configurations (#652)
bepronetwork/web-network: docs/DEPLOY-SMART-CONTRACT.md

## 1. DEPLOY SMART-CONTRACT

Fill in the `governor` wallet private key and address.

```
# Deploy Contract
DEPLOY_OWNER_ADDRESS=
DEPLOY_PRIVATE_KEY=

```

> Then it's able use a already contract deployed or deploy a new one, in this step pay attention because the **order to run database migrations** is differente in each option.

Reviewer: ```suggestion
Then will be able to use an already deployed contract or deploy a new one, in this step pay attention because the **order to run database migrations** is differente in each option.
```

### Option A - Use Existing Contract

> 1. Just update the .env file with respective addresses.

Reviewer: ```suggestion
1. Just update the .env file with the respective addresses.
```

```text
# .env

NEXT_PUBLIC_CONTRACT_ADDRESS=
NEXT_PUBLIC_SETTLER_ADDRESS=
NEXT_PUBLIC_TRANSACTION_ADDRESS=
NEXT_PUBLIC_NETWORK_REGISTRY_ADDRESS=
NEXT_PUBLIC_NFT_ADDRESS=
NEXT_PUBLIC_NFT_URI=
```

2. Run migrations.

```bash
$ npm run migrate
```

### Option B - Deploy New Contract

> Before run the script, it's able to configure some aspects for deploy, like `staging-accounts` visiting `./scripts/staging-accounts.js` or the default values and tokens, visiting `./scripts/deploy-v2.js`

Reviewer: ```suggestion
Before run the script, it's possible to configure some aspects for deploy, like `staging-accounts` visiting `./scripts/staging-accounts.js` or the default values and tokens, visiting `./scripts/deploy-v2.js`
```
